handbook/company/pricing-features-table.yml

```diff
@@ -106,7 +106,7 @@
       comingSoon: false
     - name: Programmable audit log
       tier: Premium
-      comingSoon: true
+      comingSoon: false
```

Comment:
Ah this is a good find. Thanks! I updated our C&C agenda to include double checking pricing page and transparency page changes to help us catch this earlier.

Reply:
Good idea! @zhumo when you get the chance, can you please approve and merge? Thanks :)

Reply:
I have approved, but I cannot merge. It requires @mikermcneil to approve before it gets merged.